bluemira/codes/openmc/solver.py

```diff
@@ -49,7 +44,7 @@ class OpenMCRunModes(BaseRunMode):
     """OpenMC run modes"""
 
     RUN = openmc.settings.RunMode.FIXED_SOURCE.value
-    RUN_AND_PLOT = auto()
+    RUN_AND_PLOT = "run and plot"
```

Comment:
We could change this to possibly be a Flag... see https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/5d55ea28b1203960a2548b7f7866012f6bc5125c/bluemira/equilibria/diagnostics.py#L59. Would still need to inherit from BaseRunMode.

Dont change yet but something to look at.